client/ayon_hiero/api/otio/hiero_export.py

```diff
@@ -133,7 +133,8 @@ def create_otio_reference(clip):
 
     # get file info for path and start frame
     file_info = media_source.fileinfos().pop()
-    frame_start = file_info.startFrame()
+    start_frame = file_info.startFrame()
+    frame_start = media_source.timecodeStart()
```

Comment:
lets rename it please to `timecode_start`. It might be quite confusing to read later in code.

Reply:
Sure, this is done ! Cheers.